renderer/viewer/three/graphicsBackend.ts

```
@@ -53,6 +53,61 @@
 
+// Add global waypoint testing functions for easier testing
+if (typeof window !== 'undefined') {
+  window.testWaypoints = () => {
+    if (worldRenderer?.waypoints) {
+      worldRenderer.waypoints.testWaypoint()
+      console.log('✅ Test waypoints added! Move around to see the improved directional arrows and text clarity.')
+    } else {
+      console.log('❌ World renderer not ready yet. Please wait for the world to load.')
+    }
+  }
+  
```

ESLint

[error] Trailing spaces not allowed.

(@stylistic/no-trailing-spaces)